packages/ketcher-react/src/script/ui/views/components/MonomerCreationWizard/components/NaturalAnaloguePicker/NaturalAnaloguePicker.tsx

```diff
@@ -0,0 +1,145 @@
+import { useCallback, useMemo, FC } from 'react';
+import MuiSelect, { SelectChangeEvent } from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import { MenuProps } from '@mui/material/Menu';
+import clsx from 'clsx';
+import styles from './NaturalAnaloguePicker.module.less';
+import NaturalAnalogueChip from '../NaturalAnalogueChip/NaturalAnalogueChip';
+import { Icon } from 'components';
+import { KetMonomerClass } from 'ketcher-core';
+
+interface ChipGridSelectProps {
+  monomerType: KetMonomerClass | undefined;
+  value: string;
+  onChange: (value: string) => void;
+  className?: string;
+}
+
+const ChevronIcon = ({ className }) => (
+  <Icon name="chevron" className={className} />
+);
+
+const aminoAcidOptions = [
+  { value: 'A', label: 'A', color: '#008080' },
+  { value: 'C', label: 'C', color: '#BF00FF' },
+  { value: 'D', label: 'D', color: '#FF8C69' },
+  { value: 'E', label: 'E', color: '#DC143C' },
+  { value: 'F', label: 'F', color: '#008A00' },
+  { value: 'G', label: 'G', color: '#BDB76B' },
+  { value: 'H', label: 'H', color: '#007FFF' },
+  { value: 'I', label: 'I', color: '#4CBB17' },
+  { value: 'K', label: 'K', color: '#B0E0E6' },
+  { value: 'L', label: 'L', color: '#7FFF00' },
+  { value: 'M', label: 'M', color: '#FFF600' },
+  { value: 'N', label: 'N', color: '#800080' },
+  { value: 'O', label: 'O', color: '#2A52BE' },
+  { value: 'P', label: 'P', color: '#D2D900' },
+  { value: 'Q', label: 'Q', color: '#EDB4ED' },
+  { value: 'R', label: 'R', color: '#0A12FF' },
+  { value: 'S', label: 'S', color: '#9966CC' },
+  { value: 'T', label: 'T', color: '#FF00FF' },
+  { value: 'U', label: 'U', color: '#CA7DE3' },
+  { value: 'V', label: 'V', color: '#FFD700' },
+  { value: 'W', label: 'W', color: '#CCCBD6' },
+  { value: 'X', label: 'X', color: '#CCCBD6' },
+  { value: 'Y', label: 'Y', color: '#D65CBC' },
```

Comment:
Looks like we already have these colors somewhere and we should use them from one place

Reply:
Please check packages/ketcher-macromolecules/src/components/macromoleculeProperties/MacromoleculePropertiesWindow.tsx:311

Reply:
Yes, that's exactly where I copied it from :)
That map is a macromolecules-scoped only, also it uses different structure rather than this one which is consumed by the select component. I will think of the way how I could gather it without completely duplicating the color values